src/Microsoft.ML.Tokenizers/Model/Tiktoken.cs

```diff
@@ -426,5 +494,50 @@ public override int CountTokens(string sequence, bool isSpecialToken)
         /// Gets a trainer object to use in training the model.
         /// </summary>
         public override Trainer? GetTrainer() => throw new NotImplementedException();
+
+        private static unsafe int GetUtf8Bytes(ReadOnlySpan<char> source, Span<byte> destination)
+        {
+#if NETCOREAPP
```

Comment:
We already have .netstandard and .netcoreapp cs files in the project. Will be good to move this code into these twp files and avoid using #if.